charts/seed-bootstrap/templates/grafana/ingress-auth-secret.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: grafana-ingress-credentials-basic-auth
```

Comment:
The logging secret is called `kibana-basic-auth`, so please call this `grafana-basic-auth`